inference-engine/tests_deprecated/functional/shared_tests/transformations/pooling_test.cpp

```diff
@@ -6,8 +6,12 @@
 
 std::string PoolingTestModel::getModel(SingleLayerTransformationsTestParams& p) const {
     size_t type_size = sizeof(InferenceEngine::PrecisionTrait<InferenceEngine::Precision::FP32>::value_type);
-    if (p._network_precision == "FP16")
+    netPrecision = Precision::FP32;
+
+    if (p._network_precision == "FP16") {
```

Comment:
Looks like these changes should be changed and shouldn't use network precision.
Can you add TODO in order to don't forget about it?

Reply:
added `TODO`